shm/include/player.hpp

```diff
@@ -7,11 +8,16 @@ class Player {
 public:
     Player(std::unique_ptr<Ship> ship, size_t money, size_t availableSpace);
 
-    size_t getAvailableSpace_() const { return availableSpace_; }
+    size_t getAvailableSpace() const { return availableSpace_; }
     size_t getMoney() const { return money_; }
     size_t getSpeed() const { return ship_->getSpeed(); }
     Cargo* getCargo(size_t index) const;
 
+    void setMoney(size_t money) { money_ = money; }
+
+    void loadShip(std::shared_ptr<Cargo> cargo);
+    void unloadShip(Cargo* cargo);
```

Comment:
Why do you use std::shared_ptr in loadShip and raw ptr in unloadShip?

Reply:
More comfortable would be to use shared_ptr in both places.
In task specification was proposed to use that signatures of functions to learn how to work with APIs that take raw pointers. As far as I know.